docs/understanding-packages.md

```diff
@@ -0,0 +1,228 @@
+# Understanding Packages
+
+`frock` is a CLI tool that's designed to help you with your projects; this makes
+it akin to tools like [grunt][] and [gulp][], where it uses a project-local
+configuration file, but expects to be installed in your `PATH` so that it can
+be run from your command line.
+
+This problably makes you think that `frock` should be installed *globally*, e.g.
+with an `npm install -g`, but this is *not* the best configuration for frock:
+you should instead be installing it as a `devDependency` into your project. If
```

Comment:
providing `npm install --save-dev` as a counterexample to the global command might be good here